htdocs/modules/system/preloads/core.php

```
@@ -43,6 +43,99 @@ public static function eventCoreIncludeFunctionsRedirectheader($args)
+                    __LINE__
+                );
+            }
+        } catch (\Throwable $e) {
+            $GLOBALS['xoopsLogger']->handleError(
+                E_USER_WARNING,
+                'Online tracking failed: ' . $e->getMessage(),
+                $e->getFile(),
+                $e->getLine()
+            );
+        }
+    }
```

PHPMD (2.15.0)

[warning] The method eventCoreIncludeCommonEnd() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)